app/views/ontologies/sections/metadata/_ontology_description_section.html.haml

```diff
@@ -54,7 +54,7 @@
       - l.row do
         = render FieldContainerComponent.new(label: t("ontologies.sections.metadata.categories_and_subjects")) do
           = horizontal_list_container(show_ontology_domains(domains).uniq) do |v|
-            = render ChipButtonComponent.new(text: show_category_name(v), type: "static")
+            = render ChipButtonComponent.new(text: v.split('/').last.titleize, tooltip: show_category_name(v),  type: "static")
```

Comment:
@syphax-bouazzouni 
in the method show_category_name,
when the item is not identified as a Category I propose to display the domain in the tooltip instead of a non sense acronym like C3059 ...

Reply:
what do you mean by displaying the domain, the full URL you mean?

Reply:
Yes

Reply:
Done 
![image](https://github.com/user-attachments/assets/fef1ce72-ae79-44e2-aa3c-e5c6ebf63822)

Reply:
Pass it as an <a/> tag to be clickable, cause it's difficult to copy texts from tooltip